### PR TITLE
remove some oc cluster up factory deps

### PR DESCRIPTION
**Pull request by deads2k**
Stop using the factory to build clients.  This builds on the template instantiate removal.

/assign @mfojtik 
/assign @soltysh

### CODE REVIEW

File: pkg/oc/bootstrap/clusterup/componentinstall/apply_template.go [hunk 1]
```diff
@@ -133,7 +134,7 @@ func toParamFile(params map[string]string) []byte {
 	output := ""
 
 	for k, v := range params {
-		output = output + fmt.Sprintf("%v=%v\n", k, v)
+		output = output + fmt.Sprintf("%v=%q\n", k, v)
```
Comment (mfojtik) [inline pkg/oc/bootstrap/clusterup/componentinstall/apply_template.go]: won't %q mess up the escaping?

File: pkg/oc/bootstrap/clusterup/componentinstall/readiness_apigroup.go [hunk 2]
```diff
@@ -18,62 +18,66 @@ import (
...
+			rawDiscoveryUrls = append(rawDiscoveryUrls, "/apis/"+apiService.Spec.Group+"/"+apiService.Spec.Version)
 		}
+	}
+	if len(unready) > 0 {
+		glog.V(3).Infof("waiting for readiness: %#v\n", unready)
+		return false, nil
+	}
+	// we can reasonably expect to find at least one hit.  Hit this edge with the operator.
+	if foundSet := sets.NewString(found...); !reflect.DeepEqual(requiredNames, foundSet) {
+		glog.V(3).Infof("missing: %v", requiredNames.Difference(foundSet).List())
```
[the remark this answers is not in the document]
Comment (deads2k) [reply pkg/oc/bootstrap/clusterup/componentinstall/readiness_apigroup.go]: > can we add a little bit more context here? like what is missing :-)

That's what the `requiredNames.Difference` does

File: pkg/oc/bootstrap/clusterup/componentinstall/apply_template.go (hunk 1 above)
Comment (deads2k) [reply pkg/oc/bootstrap/clusterup/componentinstall/apply_template.go]: > won't %q mess up the escaping?

Escapes are handled here: https://github.com/openshift/origin/blob/master/vendor/github.com/joho/godotenv/godotenv.go#L278

**Comment by deads2k** [comment]
@mfojtik I can't find the install container logs in the artifact logs here: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/19114/test_pull_request_origin_extended_clusterup/694/

File: pkg/oc/bootstrap/docker/openshift/util.go [hunk 3]
```diff
@@ -1,81 +0,0 @@
-package openshift
```
Comment (soltysh) [inline pkg/oc/bootstrap/docker/openshift/util.go]: One fairy saved thanks to this removal :wink:

**Pull request merged**